MIP/mip-n/README.md

```diff
@@ -0,0 +1,53 @@
+# MIP-42: AB-FFS Governed Gas Pool
+- **Description**: Introduces a governed gas pool for AB-FFS which allows for initial period wherein used gas is not recirculated.
+- **Authors**: [Liam Monninger](mailto:liam@movementlabs.xyz)
+- **Reviewer**: Andreas Penzkofer
+- **Desiderata**: [MD-38](https://github.com/movementlabsxyz/MIP/pull/38)
+
+## Abstract
+We propose a governed gas pool both as provisional and potentially permanent solution to the supply problem described in [MD-38](https://github.com/movementlabsxyz/MIP/pull/38). This solution consists of depositing funds into a pool on the L2, and holding votes against actions for the recirculation of said funds.
+
+The concept of a governed gas pool can be applied generally to manage migrations between systems. For example, the governed gas pool could be used to transition from the "Trickle-back" mechanism referenced in [MD-38](https://github.com/movementlabsxyz/MIP/pull/38) to [MIP-42](https://github.com/movementlabsxyz/MIP/pull/42), as is used as an example amongst others herein.
```

Comment:
```suggestion
The concept of a governed gas pool can be applied generally to manage migrations between systems. For example, the governed gas pool could be used to transition from the "Trickle-back" mechanism referenced in [MD-38](https://github.com/movementlabsxyz/MIP/pull/38) to [MIP-42](https://github.com/movementlabsxyz/MIP/pull/42), as is used as an example amongst others herein. As demonstrated in the diagram, the governed gas pool can be used to provide liquidity for a variety of protocol operations, liquidity of L1 Rewards, Trickle Back liquidity or grants. To use the funds for any of these activities a proposal would have to pass and a network upgrade or migration would also have to occur after governance approves the proposal. It is recommended to read generally around [aptos-governance](https://aptos.web3doc.top/concepts/governance/) and how it is implemented. Because of its tie to aptos-staking (which we do not use), we will be required to make modifications to `aptos_governance.move`. Changes for this are suggested and laid in MIP(##).
```